arm_wiki/Alternate-Install-TrueNAS.md

```diff
@@ -0,0 +1,138 @@
+# TrueNAS Installation Guide
+
+> 
+> [!Important]
+> This installation method is not supported, maintained or tested by the ARM Developers during upgrades.
+
+This is a step-by-step walkthrough adding ARM (Automatic Ripping Machine) to a
+TrueNAS Scale system as a custom app.
+
+**Note**: The following guide was performed on TrueNAS Scale Cobie (24.04).
+Prior versions of TrueNAS Scale have an issue with GPU Allocation, which Cobia fixes.
+
+
+1. **Create App**
+
+    Create a custom app via Apps → Discover Apps → Custom App 
+
+    > [!TIP]
```

Comment:
Unfortunately these [boxes won't work when nested](https://github.com/orgs/community/discussions/16925#:~:text=player%20and%20footnotes.-,Prevent%20alerts%20from%20being%20nested%20within%20other%20elements.,-The%20initial%20syntax) in the list. So they can't be indented to be part of the list. Not sure what the point of that limitation is but oh well.

I only learned that after pushing to my repo as well, had to fix it afterwards :joy: